src/app/containers/Text/index.test.jsx

```diff
@@ -44,7 +44,7 @@ describe('TextContainer', () => {
     describe('with a passed previous block type', () => {
       shouldShallowMatchSnapshot(
         'should render correctly',
-        <TextContainer {...data} typeOfPreviousBlock="text" />,
+        <TextContainer {...data} />,
```

Comment:
This whole test can be removed now, as we're no longer passing a "previous block type".

But I don't know enough about the implications of removing it - do we ever want to pass a different type of previous block? (e.g. Image?)

Reply:
I think it's ok to go now and if a prev type of block (e.g. Image) was to be accounted for, it would be an issue with design considerations. So we'd be prematurely optimising.

Reply:
Ok, agreed 👍 we should just remove this test then (as it's not testing what it says it's testing)

Reply:
I agree with Alison. We don't have any usecases for looking at previous block types at the moment. If we need to in future, we can look at implementing it at that point.

Reply:
Removed now @ChrisBAshton